Tests/TuistCoreTests/Graph/GraphTraverserTests.swift

```diff
@@ -4031,7 +4031,17 @@ final class GraphTraverserTests: TuistUnitTestCase {
                 primaryBinaryPath: "/xcframeworks/transitive.xcframework/transitive",
                 binaryPath: "/xcframeworks/transitive.xcframework/transitive"
             ),
-        ])
+            .xcframework(
```

Comment:
I don't believe this is quite right 🤔 the assertion was correct before the changes. 

See https://github.com/tuist/tuist/pull/5412#discussion_r1328900851 for the original